forte/data/readers/classification_reader.py

```diff
@@ -0,0 +1,245 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import csv
+import importlib
+from typing import Iterator, Tuple, List
+from forte.common import Resources, ProcessorConfigError
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.data.base_reader import PackReader
+from ft.onto.base_ontology import *
+
+
+__all__ = ["ClassificationDatasetReader"]
+
+
+class ClassificationDatasetReader(PackReader):
+    r"""class:`ClassificationDatasetReader` is designed to read text
+    classification dataset that contains input text and digit/text labels.
+
+    User must specify `forte_data_fields` which is a list of forte ontologies
+    or labels and it should be one-to-one correspondence with the original data
+    fields. This configuration helps generating labels and initializing data
+    fields within corresponding ontologies wrappers.
+    For example, for amazon polarity dataset,
+    https://huggingface.co/datasets/amazon_polarity, the original data fields
+    are [label, title, content]. We want to identify them as either class
+    label or specific ontologies by specifying
+    "forte_data_fields: ['label', 'ft.onto.base_ontology.Title',
+    'ft.onto.ag_news.Description']" in the configuration.
+
+    User must also specify `input_ontologies` that will be concatenate as
```

Comment:
1. the name `input_ontologies` is not intuitive, this is the field for text, so maybe something like `text_fields`
2. why this value is mandatory, will it be easier if we use all the fields from `forte_data_fields` by default?

Reply:
By default, we can use all the fields from `forte_data_fields` except for labels. Since both order and selected data fields are important for building correct input text, I didn't implement the default `forte_data_fields`. But I will add it now.